### PR TITLE
Theme: Mellow

### PR DESCRIPTION
**Pull request by kvrohit**
A brand new color scheme for helix (and other editors).

### Preview

Screenshot 2022-11-16 at 12 24 05 PM

### CODE REVIEW

File: runtime/themes/mellow.toml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+# used for lifetimes
+"label" = "yellow"
+
+"markup.heading.marker" = { fg = "gray06" }
+"markup.heading.1" = { fg = "bright_blue", modifiers = ["bold"] }
+"markup.heading.2" = { fg = "bright_blue", modifiers = ["bold"] }
+"markup.heading.3" = { fg = "bright_blue", modifiers = ["bold"] }
+"markup.heading.4" = { fg = "bright_blue", modifiers = ["bold"] }
+"markup.heading.5" = { fg = "bright_blue", modifiers = ["bold"] }
+"markup.heading.6" = { fg = "bright_blue", modifiers = ["bold"] }
```
Comment (archseer) [inline runtime/themes/mellow.toml:54]: `markup.heading` should be enough

Comment (kvrohit) [reply runtime/themes/mellow.toml]: Updated!

File: runtime/themes/mellow.toml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+"ui.virtual" = { fg = "gray02" }
+"ui.virtual.indent-guide" = { fg = "gray02" }
+
+"ui.selection" = { bg = "gray03" }
+"ui.selection.primary" = { bg = "gray03" }
+
+"ui.cursor" = { bg = "gray04" }
+"ui.cursor.insert" = { bg = "gray04" }
+"ui.cursor.match" = { fg = "yellow", modifiers = ["bold", "underlined"] }
+"ui.cursor.select" = { bg = "gray04" }
```
Comment (the-mikedavis) [inline runtime/themes/mellow.toml:89]:
```suggestion
"ui.cursor" = { bg = "gray04" }
"ui.cursor.match" = { fg = "yellow", modifiers = ["bold", "underlined"] }
```

Those will fall back to `ui.cursor` if not defined

**Comment by kvrohit** [comment]
Updated highlights for markdown headings and UI cursor as per suggestions from code reviews. Thanks.

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Comment by kvrohit** [comment]
Screenshot added: https://github.com/helix-editor/helix/wiki/Themes#mellow. Thanks for the merge.

**Pull request merged**